app/routes/events/components/EventDetail/index.js

```diff
@@ -460,21 +476,6 @@ export default class EventDetail extends Component<Props, State> {
               </Flex>
             </ContentSidebar>
           </ContentSection>
-          <Link
-            to="/pages/arrangementer/26-arrangementsregler"
-            style={{ marginTop: 0 }}
-          >
-            <Flex alignItems="center">
-              <Icon name="document" style={{ marginRight: '4px' }} />
-              <span>Regler for Abakus&#39; arrangementer</span>
```

Comment:
> I would actually prefer to have "the rules" where it already was, as it's preferable if the sidebar isn't too crowded (ref. the PR we just merged addressing this very issue).

I agree that it's important to keep it light, but I've talked to people that weren't even aware that we have event rules - so I think the place where it is now is way to hidden, and I don't know any other places where it would fit, so I'd like to keep it in the bottom of the sidebar.

Reply:
I'll supplement that [we now have](https://github.com/webkom/lego-webapp/pull/2682) a tooltip on the "Frist for prikk" containing a link to the rules page. I'd argue the page is mostly used in regards to the "prikk" logic, and with the (very visible in my opinion) tooltip, this addition becomes less important, don't you think?

Reply:
Good point. With the updated design I put it in the bottom of the sidebar, so it's practically in the same place as it currently is on mobile (except for elevated users) - but a bit more visible on desktop, and shortened it down as the point of it including "prikker" is already stated in the prikkefrist tooltip.

Reply:
It's much clearer now! What do you think about using the outlined document icon instead of the default filled one? Wouldn't that match the others more? 😄 Yes, I do realize it was filled from before.

Reply:
Also, the icons used on this page (including the follow star) are `ios-` icons, but, in my opinion the `md-` ones are much prettier. Just a though given, and I know it's out of scope of what you're trying to accomplish with this PR!

Reply:
Agree that they should match, frankly the thought that document-outline existed never hit me :o  
Updated to both being outlined now